debuginfo/src/dwarf.rs

```diff
@@ -404,6 +404,16 @@ impl<'d, 'a> DwarfUnit<'d, 'a> {
             None => return Err(gimli::read::Error::MissingUnitDie.into()),
         };
 
+        // Clang's LLD might eliminate an entire compilation unit and simply set the low_pc to zero
+        // and remove all range entries to indicate that it is missing. Skip such a unit, as it does
+        // not contain any code that can be executed.
```

Comment:
Is it worth explaining the `info.kind` part of the test, by noting that a `.o` file might legitimately have a zero address?

Reply:
Absolutely, thanks for the suggestion!